README.md

```diff
@@ -44,53 +73,145 @@ If you want to invite the public Winnie_Bot account to your server, go [here](ht
+  </li>
+  <li>
+    Create a <a href="https://docs.github.com/en/free-pro-team@latest/desktop/contributing-and-collaborating-using-github-desktop/creating-an-issue-or-pull-request#creating-a-pull-request">pull request</a> back to Winnie with your changes.
+  </li>
+</ol>
 
-* **Dawn E. Collett** - *Maintainer* - [GitHub](https://github.com/lisushka)
-* **Katie Macke** - *Development Team* - [GitHub](https://github.com/asleepysheepy)
-* **Robert W. McLeod** - *Server management assistance* - [GitHub](https://github.com/RobFaie)
-* **Lauren Jenkinson** - *Server management assistance* - [GitHub](https://github.com/alxce)
+#### Translation files explained
+
+* `winnie.json` - Miscellanous translations that don't better fit in another file
+* `commands.json` - Command response messages
+
+### Developing Winnie
+
+#### Requirements
+* [Docker](https://www.docker.com/)/[Docker Compose](https://docs.docker.com/compose/)
+* [Node.js](https://nodejs.org/en/) 14.0 or higher
+* [Yarn](https://yarnpkg.com/)
+
+#### Running Winnie for local development
+
+<ol>
+  <li>
+    clone the repo and move into the directory
+    <br>
+    <pre>$ git clone https://github.com/aigeroni/Winnie_Bot.git winnie_bot && cd winnie_bot</pre>
+  </li>
+  <li>
+    Install dependencies with yarn 
+    <br>
+    <pre>$ yarn install</pre>
+  </li>
+  <li>
+    Create <code>.env</code> file
+    <br>
+    <pre>$ cp .env.sample .env</pre>
+  </li>
+  <li>
+    Setup your evironment variables in the <code>.env</code> file.
+  </li>
+  <li>
+    Start the docker processes 
+    <br>
+    <pre>$ yarn docker:local:up -d</pre>
+  </li>
+  <li>
+    Run the mirgations to set up the database
+    <br>
+    <pre>$ yarn typeorm migration:run</pre>
+  </li>
+  <li>
+    Start Winnie
+    <br>
+    <pre>$ yarn start</pre>
+  </li>
+</ol>
+
+#### package.json scripts
+
+* `yarn build` - Compiles Winnie's typescript code into plain JavaScript
+* `yarn docker:local:down` - Teardown local docker processes
+* `yarn docker:local:up` - Start up local docker processes
+* `yarn lint:all` - Run eslint over Winnie's code
+* `yarn lint:fix` - Run eslint over Winnie's code, ficing mistakes
+* `yarn start` - Compile Winnie and start up the bot process
+* `yarn typeorm` - Access Typeorm's CLI
+
+## Core Team
+
+| Dawn E. Collett | Jason E. Gillikin | Katie Macke |
+|---|---|---|
+| [![Dawn's Github](https://avatars1.githubusercontent.com/u/28942094?s=100&v=4)](https://github.com/lisushka) | [![Jason's Github](https://avatars2.githubusercontent.com/u/7763031?s=100&v=4)](https://github.com/jegillikin) | [![Katie's Github](https://avatars1.githubusercontent.com/u/12132647?s=100&v=4)](https://github.com/asleepysheepy) |
 
 ## License
 
-This project is licensed under the GNU General Public License, v3.0 - see the [LICENSE](LICENSE) file for details.  In particular, if you incorporate Winnie's source code into another project, you must release the source code of that project.

```

Comment:
Can we add the line about needing to release your source code under GPL back in?

Reply:
whoops, didn't even know that was there